velox/docs/develop/operators.rst

```diff
@@ -291,6 +292,125 @@ Note: Presto allows multiple global grouping sets in a single SQL query.
 
 Hence, globalGroupingSets is a vector of groupIds.
 
+.. _ExpandNode:
+
+ExpandNode
+~~~~~~~~~~~
+
+For each input row, generates N rows with M columns according to specified 'projections'.
+'projections' is an N x M matrix of expressions: a vector of N rows each having M columns.
+Each expression is either a column reference or a constant. Both null and non-null constants are allowed.
+'names' is a list of M new column names. The semantic of this operator matches Spark. Using project and unnest can be
+employed to implement the expand functionality. However, the performance is suboptimal when creating an array
+constructor within the Project operation.
+
+.. list-table::
+   :widths: 10 30
+   :align: left
+   :header-rows: 1
+
+   * - Property
+     - Description
+   * - projections
+     - A vector of N rows each having M columns. Each expression is either a column reference or a constant.
+   * - names
+     - A list of new column names.
+
+ExpandNode is typically used to compute GROUPING SETS, CUBE, ROLLUP and COUNT DISTINCT.   
+
+To illustrate how ExpandNode works lets examine the following SQL query:
+
+.. code-block:: sql
+
+  SELECT l_orderkey, l_partkey, sum(l_suppkey) FROM lineitem GROUP BY ROLLUP(l_orderkey, l_partkey);
```

Comment:
> sum(l_suppkey)

This example is a bit strange as sum(suppkey) doesn't have any obvious meaning. Perhaps, change to sum(quantity), count(1) or similar.

Reply:
@mbasmanova  Changed to `count(l_suppkey)`.